admin/resources/js/components/poolCandidate/UpdatePoolCandidate.tsx

```diff
@@ -54,11 +58,16 @@ type FormValues = Pick<
   | "expectedSalary"
   | "locationPreferences"
   | "status"
-> & {
-  acceptedOperationalRequirements: string[] | undefined;
-  cmoAssets: string[] | undefined;
-  expectedClassifications: string[] | undefined;
-};
+> &
+  Pick<
+    User,
+    "email" | "firstName" | "lastName" | "telephone" | "preferredLang"
+  > & {
+    userId: Scalars["ID"]; // can't use pick since we need a new name
```

Comment:
I haven't found a way to rename the id through the Pick. However, instead of using `Scalars["ID"]`, we can change the type to `userId: User["id"];`. I guess it would make it a little clearer.

Reply:
I'll add a suggestion.